workers/data_refinery_workers/processors/create_quantpendia.py

```diff
@@ -124,12 +120,25 @@ def create_result_objects(job_context: Dict) -> Dict:
     return job_context
 
 
+def remove_job_dir(job_context: Dict):
+    """ remove the directory when the job is successful. At this point
+    the quantpendia was already zipped and uploaded. """
+    shutil.rmtree(job_context["job_dir"], ignore_errors=True)
+    return job_context
+
+
+def get_process_stats():
+    BYTES_IN_GB = 1024 * 1024 * 1024
+    process = psutil.Process(os.getpid())
+    ram_in_GB = process.memory_info().rss / BYTES_IN_GB
+    return { 'total_cpu': psutil.cpu_percent(), 'process_ram': ram_in_GB }
+
 def _make_dirs(job_context: Dict):
     dataset_id = str(job_context["dataset"].pk)
-    job_context["work_dir"] = "/home/user/data_store/smashed/" + dataset_id + "/"
-    os.makedirs(job_context["work_dir"])
-    job_context["output_dir"] = job_context["work_dir"] + "output/"
-    os.makedirs(job_context["output_dir"])
+    job_context["job_dir"] = "/home/user/data_store/smashed/" + dataset_id + "/"
+    os.makedirs(job_context["job_dir"], exist_ok=True)
+    job_context["output_dir"] = job_context["job_dir"] + "output/"
+    os.makedirs(job_context["output_dir"], exist_ok=True)
```

Comment:
```suggestion
    os.makedirs(job_context["output_dir"], exist_ok=True)
    
    return job_context
```
This goes with the other comment above.